wsclient/src/main/java/org/hpccsystems/ws/client/utils/Utils.java

```diff
@@ -64,14 +68,24 @@ public static String parseVersionFromWSDLURL(String wsdlurl)
 
     static public void println(PrintStream stream, String message, boolean onlyifverbose, boolean verbosemode )
     {
-        if (verbosemode || !onlyifverbose)
-            stream.println(message);
+        // if (verbosemode || !onlyifverbose)
+        //     stream.println(message);
+        if (onlyifverbose) {
+            log.debug(message);
+        } else {
+            log.error(message);
```

Comment:
this might need to target log.trace

Reply:
Are you saying use debug & trace or error & trace?